### PR TITLE
Feature toggle temp

### PR DESCRIPTION
**Pull request by Oxman345**
forecast isnt displaying but logic is there for the toggle

### CODE REVIEW

File: weather-app/src/App.js [hunk 1]
```diff
@@ -1,57 +1,99 @@
-import React, { useState } from 'react';
+import React, { useState } from "react";
+import { TextField } from "@material-ui/core";
 
-import CurrentCondition from "./CurrentCondition";
-import Forecast from "./Forecast";
+import Weather from "./Weather";
+// import CurrentCondition from "./CurrentCondition";
+// import Forecast from "./Forecast";
```
Comment (Oxman345) [inline weather-app/src/App.js:6]: remove

File: weather-app/src/App.js [hunk 2]
```diff
@@ -1,57 +1,99 @@
...
+    if (e.key === "Enter" && e.target.value !== "") {
+      fetch(`${api}&q=${query}&days=3`)
+        .then((res) => res.json())
+        .then((result) => {
+          setAlert(false);
+          setCurrent(result.current);
+          setForecast(result.forecast.forecastday);
+          setLocation(result.location);
+          setQuery("");
+          console.log(result);
```
Comment (Oxman345) [inline weather-app/src/App.js:27]: remove

File: weather-app/src/App.js [hunk 3]
```diff
@@ -1,57 +1,99 @@
...
+        })
+        .catch((error) => {
+          console.log(error);
```
Comment (Oxman345) [inline weather-app/src/App.js:30]: remove

File: weather-app/src/App.js [hunk 4]
```diff
@@ -1,57 +1,99 @@
...
+          </div>
         </div>
-       </>
-     );
-
+        {location && forecast && current && (
+          <Weather location={location} forecast={forecast} current={current} />
+        )}
+        {/*{" "}
+        {current && (
```
Comment (Oxman345) [inline weather-app/src/App.js:73]: remove

File: weather-app/src/Forecast.js [hunk 5]
```diff
@@ -1,45 +1,73 @@
...
+          {props.isFarenheit ? (
+            <div>{props.day.mintemp_f}°F</div>
+          ) : (
+            <div>{props.day.mintemp_c}°C</div>
+          )}
+        </div>
+        <div>
+          <strong>Humidity </strong> {props.day.avghumidity}
+        </div>
+        {/* <div style={{ display: "flex", flexDirection: "column" }}>
```
Comment (Oxman345) [inline weather-app/src/Forecast.js:60]: remove

File: weather-app/src/Weather.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+import Forecast from "./Forecast";
+
+function Weather(props) {
+  const current = props.current;
+  const location = props.location;
+  const forecast = props.forecast;
+  const [isFarenheit, setIsFarenheit] = useState(true);
+
+  const toggleTemp = () => {
+    console.log("Forecast ====>", props.forecast);
```
Comment (Oxman345) [inline weather-app/src/Weather.js:14]: remove

File: weather-app/src/Weather.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+      </div>
+      {current && (
+        <CurrentCondition
+          {...current}
+          city={location.name}
+          state={location.region}
+          isFarenheit={isFarenheit}
+        />
+      )}
+      {console.log(props)}
```
Comment (Oxman345) [inline weather-app/src/Weather.js:43]: remove

**Review by OpenWebGuildMaster — APPROVED** [review]
looks good to me, lets merge

**Pull request merged**